pkg/invoke/raw_exec.go

```diff
@@ -52,11 +52,7 @@ func pluginErr(err error, output []byte) error {
 		if perr := json.Unmarshal(output, &emsg); perr != nil {
 			return fmt.Errorf("netplugin failed but error parsing its diagnostic message %q: %v", string(output), perr)
```

Comment:
Do we want to generate a `types.Error` for this condition as well? Then we can tighten up the return types.